[FIX] clarify XXXCoord* in the coordinate systems appendix (#520)
bids-standard/bids-specification: src/99-appendices/08-coordinate-systems.md

hand side of the head, the second dimension (Y) points towards the Anterior
aspect of the head, and the third dimension (Z) points towards the top of the
head.

> The directions are following the *neurological convention*, so the subject's
> left side of the head is also on the left side of the image.

Reviewer: I think this is meant to say something like:

```suggestion
The directions are considered to be from the subject's perspective.
For example, in the `RAS` coordinate system, a point to the subject's left
will have a negative `x` value.
```

It is possible that others use neurological/radiological to specify this concept, but that's a conflation of coordinates and visualization. Since we don't use it elsewhere, I don't think introducing and defining it adds anything.

## Coordinate Systems applicable to MEG, EEG, and iEEG

Generally, across the MEG, EEG, and iEEG modalities, the first two pieces of
information for a coordinate system (origin and orientation) are specified in

> `<datatype>CoordinateSystem`. The third piece of information for a coordinate
> system (units) are specified in `<datatype>CoordinateSystemUnits`.

Reviewer: ```suggestion
`<datatype>CoordinateSystem`.
The third piece of information for a coordinate system (units) are specified in
`<datatype>CoordinateSystemUnits`.
```

Here, `<datatype>` can be one of `MEG`, `EEG`, or `iEEG`, depending on the
modality that is being used.

Allowed values for the `<datatype>CoordinateSystem` field come from a list of
restricted keywords, as listed in the sections below.
If no value from the list of restricted keywords fits, there is always the
option to specify the value as follows:

-   `Other`: Use this for other coordinate systems and specify further details
    in the `<datatype>CoordinateSystemDescription` field

> **If you feel like a specific coordinate system should be added to the list

Reviewer: ```suggestion
**If you believe a specific coordinate system should be added to the list
```

> of restricted keywords for either MEG, EEG, or iEEG, please open a new issue
> on the

Reviewer: ```suggestion
of restricted keywords for MEG, EEG, or iEEG, please open a new issue on the
```

## EEG Specific Coordinate Systems

Restricted keywords for the `<datatype>CoordinateSystem` field in the
`coordsystem.json` file for EEG datasets:

> -   `CapTrak`: RAS orientation and the origin between the ears

Reviewer: Looks like a backwards incompatible change. Is it just a typo being fixed? Do we need to worry about someone taking it too seriously in the past?

Author: yes, a typo fix. I haven't seen any datasets that use CapTrak yet --> and in fact, the list of restricted keywords for CoordSystems is not yet validated, see: https://github.com/bids-standard/bids-validator/issues/945

due to that missing coverage in the validator we currently have several datasets on openneuro that specify their coordsystem as e.g., "ARS", which is invalid of course. (see e.g., https://openneuro.org/datasets/ds002680/versions/1.0.0)

Reviewer 2: agreed, being specified as "ARS" we still don't know how to relate electrode positions to the anatomy of the head, whereas "Other" and some textual description could have given more information.
